apps/meteor/server/models/raw/Roles.ts

```diff
@@ -100,6 +136,30 @@ export class RolesRaw extends BaseRaw<IRole> implements IRolesModel {
 		return true;
 	}
 
+	async removeUsersRoles(uids: IUser['_id'][], roles: IRole['_id'][], scope?: IRoom['_id']): Promise<boolean> {
+		if (process.env.NODE_ENV === 'development' && (scope === 'Users' || scope === 'Subscriptions')) {
+			throw new Error('Roles.removeUserRoles method received a role scope instead of a scope value.');
+		}
+
+		for await (const roleId of roles) {
+			const role = await this.findOneById<Pick<IRole, '_id' | 'scope'>>(roleId, { projection: { scope: 1 } });
+
+			if (!role) {
+				continue;
+			}
+
+			switch (role.scope) {
+				case 'Subscriptions':
+					scope && (await Subscriptions.removeRolesByUserIds(uids, [roleId], scope));
+					break;
+				case 'Users':
+				default:
+					await Users.removeRolesByUserIds(uids, [roleId]);
+			}
+		}
+		return true;
```

Comment:
To keep this new method consistent with the other methods, you see all other similar methods are returning a boolean, so that's why I did not change it.

Reply:
I'll have to check that then, but here it doesn't make any sense. No matter what it'll always return true if iirc, otherwise it'll jump out of the stack.